icons/opencollective.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Open Collective icon</title><path d="M21.86 5.17a11.94 11.94 0 0 1 0 13.66l-3.1-3.1a7.68 7.68 0 0 0 0-7.46l3.1-3.1zm-3.03-3.03l-3.1 3.1a7.71 7.71 0 1 0 0 13.51l3.1 3.11a12 12 0 1 1 0-19.73 M21.86 5.17a11.94 11.94 0 0 1 0 13.66l-3.1-3.1a7.68 7.68 0 0 0 0-7.46l3.1-3.1z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Open Collective icon</title><path d="M21.854 5.178a11.936 11.936 0 010 13.654l-3.098-3.098a7.677 7.677 0 000-7.458zm-3.028-3.03l-3.1 3.1a7.707 7.707 0 100 13.505l3.1 3.108a11.996 11.996 0 110-19.722m3.028 3.039a11.936 11.936 0 010 13.654l-3.098-3.098a7.677 7.677 0 000-7.458z"/></svg>
```

Comment:
Looks like there's a *slight* difference between PR (red) and source (black)
![image](https://user-images.githubusercontent.com/1830867/102927557-1ba8ee00-448f-11eb-826b-d4d2da484e93.png)

Any chance of a rebuild, see if it removes the issue?

Reply:
@adamrusted  Done 👍🏼